Week-5/duplicatecheck.js

```diff
@@ -0,0 +1,23 @@
+function duplicateCheck(arr) {
+  var set = new Set();
+  for (let el of arr) {
```

Comment:
@rajeshvlega9 We don't need to iterate to fill the set. Just use as new Set(arr). It will remove all duplicates by doing so.